### PR TITLE
php-function-cosh

### LINKED ISSUE #2207: [Term Entry] PHP Math Functions cosh()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `cosh()` function.

The entry should go in a new file in `docs/content/php/concepts/math-functions/terms/cosh/cosh.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

Thanks for assigning me. This closes #2207 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/cosh/cosh.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The output will be:
+
+```shell
+6.1322894796637
+```
+
+## Codebyte Example
+
+In this executable example of the function, the `$x` variable can be changed to print the `cosh()` of it
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/cosh/cosh.md:47]:
```suggestion
In this executable example of the function, the `$x` variable can be changed to print the `cosh()` of it:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikolaspoczekaj I did a first review and only had one comment.

**Comment by nikolaspoczekaj** [comment]
@SSwiniarski Done, thanks :)

**Review by SSwiniarski — APPROVED** [review]
@nikolaspoczekaj Looks good for a second review.

File: content/php/concepts/math-functions/terms/cosh/cosh.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The **`cosh()`** function in PHP returns the hyperbolic cosine of a number. It calculates the value using the exponential function and is useful in various mathematical calculations.
+
+## Syntax
+
+```pseudo
+$result = cosh(number);
+```
+
+`number` - represents the input value for which the hyperbolic cosine will be returned.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/cosh/cosh.md:25]:
```suggestion
`number`: Represents the input value for which the hyperbolic cosine will be returned.
```

File: content/php/concepts/math-functions/terms/cosh/cosh.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+?>
+```
+
+The output will be:
+
+```shell
+6.1322894796637
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/cosh/cosh.md:45]: We don't duplicate examples in docs. I would recommend deleting the codebyte unless there is some additional functionality that is worthwhile illustrating with a different example.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj I've added a couple of comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by nikolaspoczekaj** [comment]
Thanks @caupolicandiaz . Finished making the changes.

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj @SSwiniarski  Looks good

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/math-functions/cosh

image

**Pull request merged**